.github/workflows/validation.yml

```diff
@@ -34,6 +34,8 @@ jobs:
       - run: git status
       - name: lint
         run: yarn lint
+      - name: shellcheck
```

Comment:
Can we create separate workflow for this? this validation step is very long already and this actually doesn't depends on any JS stuff

Reply:
Sure, I can create another validation (shell-validation) workflow. 
I could run these only for *.sh files.

Running this step takes ~2s though, didn't seem like a big deal.